contrib/src/main/java/com/datatorrent/contrib/cassandra/AbstractCassandraInputOperator.java

```diff
@@ -45,7 +47,8 @@
 public abstract class AbstractCassandraInputOperator<T> extends AbstractStoreInputOperator<T, CassandraStore> {
 
   private static final Logger logger = LoggerFactory.getLogger(AbstractCassandraInputOperator.class);
-
+  private PagingState nextPageState;
+  private int fetchSize;
   int waitForDataTimeout = 100;
   @AutoMetric
   protected long tuplesRead;
```

Comment:
@DT-Priyanka 
If tuples count is not right, isn't that the problem with all the operators? 
In that case why did we decide to add this user metric to just this operator?

I asked this question because I want to understand what was use case which made it necessary to add this metric just to cassandra input operator.